smoke-test/tests/utils.py

```diff
@@ -114,6 +113,9 @@ def ingest_file_via_rest(filename: str) -> Pipeline:
 
 
 def delete_urns_from_file(filename: str) -> None:
+    if not cli_utils.get_boolean_env_variable("CLEANUP_DATA", True):
+        print("Not cleaning data to save time")
```

Comment:
Interested to understand how much time data cleanup is actually taking

Reply:
For cypress tests it was taking 5 mins which seemed too much to me given we will not be doing anything after that in Github CI.

Reply:
From https://github.com/datahub-project/datahub/actions/runs/3762214471/jobs/6394937771
```
============================= slowest 20 durations =============================
2487.69s call     tests/cypress/integration_test.py::test_run_cypress
288.20s teardown tests/cypress/integration_test.py::test_run_cypress
51.26s call     test_e2e.py::test_ingestion_via_kafka
26.24s call     tests/tags-and-terms/tags_and_terms_test.py::test_update_schemafield
26.20s teardown tests/containers/containers_test.py::test_update_container
22.56s call     test_e2e.py::test_native_user_endpoints
16.10s teardown tests/views/views_test.py::test_update_global_view
14.51s call     tests/cli/delete_cmd/test_timeseries_delete.py::test_timeseries_delete
13.28s setup    tests/cypress/integration_test.py::test_run_cypress
12.27s call     tests/containers/containers_test.py::test_update_container
12.23s call     tests/cli/datahub_graph_test.py::test_graph_relationships
12.12s call     tests/managed-ingestion/managed_ingestion_test.py::test_create_list_get_remove_ingestion_source
12.11s call     tests/managed-ingestion/managed_ingestion_test.py::test_create_list_get_remove_secret
12.11s call     tests/domains/domains_test.py::test_create_list_get_domain
10.46s call     test_e2e.py::test_ingestion_via_rest
10.28s call     test_e2e.py::test_add_remove_members_from_group
10.13s call     tests/policies/test_policies.py::test_frontend_policy_operations
10.10s call     tests/managed-ingestion/managed_ingestion_test.py::test_create_list_get_ingestion_execution_request
10.09s call     tests/views/views_test.py::test_create_list_delete_global_view
10.09s call     tests/views/views_test.py::test_create_list_delete_personal_view
=========== 1 failed, 132 passed, 20 warnings in 3524.49s (0:58:44) ============

```